{{cookiecutter.project_slug}}/_/deployment/python/deployment/kubernetes/application/base/route.yaml

```diff
@@ -0,0 +1,15 @@
+kind: Route
+apiVersion: v1
```

Comment:
Let's always put the `apiVersion` first, to be consistent

Reply:
`apiVersion` is always first now.